packages/code-editor/src/CodeEditor/CodeEditor.types.ts

```diff
@@ -123,4 +163,9 @@ export interface CodeEditorProps extends DarkModeProps {
    * Enables read only mode, making the contents uneditable.
    */
   readOnly?: boolean;
+
+  /**
+   * Add tooltips to the editor content.
+   */
+  tooltips?: Array<CodeEditorTooltip>;
```

Comment:
If we're only officially supporting one tooltip, we could potentially type this as a 1D tuple: `[CodeEditorTooltip]`